hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -104,6 +105,7 @@ public class BlockOutputStream extends OutputStream {
   private int chunkIndex;
   private final AtomicLong chunkOffset = new AtomicLong();
   private final BufferPool bufferPool;
+  private final DirectBufferPool directBufferPool;
```

Comment:
@ashishkumar50 , I would suggest reuse the directBufferPool in one Ozone client. Direct buffer allocation is expensive, one buffer allocated in one OutputStream can be reused by another OutputStream.  The simple solution is declare this as a stack field, 

```
  private static final DirectBufferPool directBufferPool = new DirectBufferPool();
```

Reply:
Used static `DirectBufferPool` in `BlockOutputStream`